bin/iso/create.js

```diff
@@ -28,9 +31,13 @@ module.exports = resource => Cli.createCommand('create', {
 
         let iso;
 
-        if (args.source.match('^http(s)?://')) {
-            iso = await args.helpers.api.post(resource.url(args), { name: args.name, source: args.source });
-        } else {
+        if(!(!args['source-url'] ^ !args['source-file'])){
```

Comment:
We need ```NOT( A XOR B )``` and there is [no logical XOR in javascript](https://stackoverflow.com/questions/4540422/why-is-there-no-logical-xor-in-javascript#4540443).

![obraz](https://user-images.githubusercontent.com/3618479/40919208-236866bc-6809-11e8-8616-320327cbdbbe.png)